ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -299,40 +305,73 @@ export default function AssetsPanel() {
       title={translations.assetsPanel}
       classes={dragInProgress ? { body: classes.noScroll } : null}
     >
-      <ErrorBoundary>
```

Comment:
No need to modify the components structure. Simply replace error boundary with the suspencified with empty state